internal/controller/api/hiveextension/v1beta1/agentclusterinstall_types.go

```diff
@@ -98,7 +98,11 @@ type AgentClusterInstallStatus struct {
 type DebugInfo struct {
 	// EventsURL specifies an HTTP/S URL that contains events which occured during the cluster installation process
 	// +optional
-	EventsURL string `json:"eventsURL,omitempty"`
```

Comment:
no need for omitempty?

Reply:
i was sure that `omitempty` and `+optional` are the same.